example/real-federated-store/have-a-go-at-operations/curl-operation-examples/curl_getAllElements.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
What's this line for? Would usually be `#!/bin/bash`. Shouldn't really be needed and not all the files have it.